inspirehep/modules/pidstore/utils.py

```diff
@@ -45,6 +45,29 @@ def get_endpoint_from_pid_type(pid_type):
     return PID_TYPE_TO_ENDPOINT[pid_type]
 
 
+def get_endpoint_from_schema(schema):
```

Comment:
I have a feeling this file is going out of hand with so many conversions back and forth... but don't know if there is a better solution for this case.